### PR TITLE
data:sharsha315

### PR DESCRIPTION
**Pull request by sharsha315**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5222"></a>

### CODE REVIEW

File: data/sharsha315.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Harsha S",
+  "displayStatsPublic": true,
```
Comment (swagataroy30) [inline data/sharsha315.json:3]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we don't need to add `"displayStatsPublic": true` anymore. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

File: data/sharsha315.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/sharsha315",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://www.github.com/sharsha315",
+      "icon": "FaGithub"
+    },
```
Comment (swagataroy30) [inline data/sharsha315.json:21]:
```suggestion
    }
```

**Review by swagataroy30 — COMMENTED** [review]
You need to remove the comma at line 21. Thank You!

**Review by SurajPratap10 — COMMENTED** [review]
Everything is correct. 

`Suggestion:` you can use `SiLinkedin` as it looks better and add more roles in your bio if you wish.

**Comment by SurajPratap10** [comment]
> everything is correct. `Suggestion:` you can use `SiLinkedin` as it looks better and if you wish then you can also add a few more roles in your bio.

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by SurajPratap10 — APPROVED** [review]
LGTM

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**